src/content/jobs/full-stack-ai-lead.md

```diff
@@ -0,0 +1,114 @@
+---
+meta:
+    title: Fullstack AI Lead - Dołącz do Bethink!
+    description: Świetny developer dla każdego - starego i młodego
+    linkImageUrl: asdfasdf
+title: Fullstack AI Lead
+excerpt: Nowoczesny stack, doświadczony zespół i praca nad real-time’ową, społecznościową aplikacją do e-learningu. Rozwijaj aplikację, na której prowadzone są największe w Polsce medyczne i maturalne kursy online.
+salaryRange: 27.5 - 37.5k brutto UOP (80% koszty uzyskania przychodu) / 27.5 - 37.5k netto B2B
+isRemote: true
+externalLink:
+    text: 'Otwórz w JustJoin.IT'
+    url: https://justjoin.it/job-offer/bethink-fullstack-ai-lead-poznan-ai
+isDraft: false
+isOpen: true
+isArchived: false
+heroImage:
+    url: asdf
+    alt: asdf
+---
+
+Nowoczesny stack, doświadczony zespół i praca nad autorską platformą e-learningową.
+
+Rozwijaj aplikację, na której prowadzone są największe w Polsce medyczne i maturalne kursy online.
+
+### Gdzie, z kim, za ile?
+
+-   📍 Praca remote lub w biurze (Poznań, Podlaska 15, Sołacz)
+-   💵 29.3 - 41.8k brutto UOP (80% koszty uzyskania przychodu) / 29.3 - 41.8k netto B2B
```

Comment:
It's inconsistent with salaryRange in frontmatter.

Reply:
removed